pixelgl/window.go

```diff
@@ -163,6 +168,7 @@ func NewWindow(cfg WindowConfig) (*Window, error) {
 		// enter the OpenGL context
 		w.begin()
 		glhf.Init()
+		gl.Enable(gl.MULTISAMPLE)
```

Comment:
Does having this enabled with a cfg.SamplesMSAA == 0 have any effect? If so, do we need a condition around this toggle?

Reply:
when the window hint for sampels is set to 0 (aka cfg.SamplesMSAA == 0), this doesn't do anything. From my reading as well, this is almost always enabled automatically by the driver whether its in use or not, so there's no reason to have a conditional.